config/samples/jobtemplate/tower_v1alpha1_jobtemplate.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: jobtemplate-sample
 spec:
   tower_auth_secret: awxaccess
-  job_template_name: ExampleJobTemplate
-  job_template_project: Demo Project
-  job_template_playbook: hello_world.yml
-  job_template_inventory: Demo Inventory
```

Comment:
These changes look good, I think the logic in there will work. But could you try adding a separate samples file here including the deprecated settings though?
https://github.com/ansible/awx-resource-operator/pull/116/files#diff-142fc39bd3063aef4cf4cd2003595dab4fe1b25e77989f1a292b26397985baa0
Maybe we call it config/samples/jobtemplate/tower_v1alpha1_jobtemplate_deprecated.yaml





The idea being that CI will exercise the deprecated variables to make sure the |default(omit) bit works for the deprecated variables
You'll need to add a line here so that CI tries to create it:
https://github.com/ansible/awx-resource-operator/blob/1dfed85a5fb25e1c74765500813a5794eaf717e6/.github/workflows/kind-awx.yaml#L168